### DISCUSSION TITLE
Bark died ☠️

### DISCUSSION THREAD

**Discussion by mariocraft987** [cytrinc/bark-coding#255] [General]
f

**Comment by TheBestUser01234567890** [comment]
I can contribute if needed.

**Reply by mariocraft987** [reply]
pull requests

**Reply by mariocraft987** [reply]
you should join our discord though

**Reply by TheBestUser01234567890** [reply]
sorry i dont use disc and i dont plan to use it either

**Reply by dumorando** [reply]
what would you contribute
![image](https://github.com/user-attachments/assets/3ca8bcd2-f6be-4688-85ad-0082076e320c)

**Reply by TheBestUser01234567890** [reply]
You forgot I also know js, I can add things like extensions and I can rewrite some code

**Reply by dumorando** [reply]
mkay just remember contributing.md

**Comment by raynepaws** [comment]
i for one will not let it die and will continue working on the project

**Reply by mariocraft987** [reply]
nice
